### ISSUE TITLE
problem with version1.4. from window10 . 

### ISSUE THREAD

**Issue by Dedalios** [videomorph-dev/videomorph#41]
hello, I have a problem for 1 week with your version1.4.
The sequence of files to convert does not work. The software hangs after processing the first file in the list.
Stopping the software no longer works.
I uninstall and rehint the problem is still present.
version: videomorph1.4-win_x86_64.exe

I have made updates to python 3.5.

I didn't have this problem 15 days ago

**Comment by leonel-lordford** [comment]
We'll review this, but for now our approach is to deliver a new version of FFmpeg and Python in the next release.

**Comment by lpozo** [comment]
@leonel-lordford, @codeshard I think this problem is closely related to issue #2 (same behavior). The Windows installer includes its own version of Python. So, changing the Python version on the system should not affect the application.

**Comment by leonel-lordford** [comment]
I agree, but since we don't use those binaries we need to assume they just work... and my proposal is to upgrade all binaries to versions similar to those  we have in GNU/Linux :-) and try it.
